ports/unix/mpconfigport.h

```diff
@@ -105,7 +105,7 @@ typedef long mp_off_t;
 // Always enable GC.
 #define MICROPY_ENABLE_GC           (1)
 
-#if !(defined(MICROPY_GCREGS_SETJMP) || defined(__x86_64__) || defined(__i386__) || defined(__thumb2__) || defined(__thumb__) || defined(__arm__))
+#if !(defined(MICROPY_GCREGS_SETJMP) || defined(__x86_64__) || defined(__i386__) || defined(__thumb2__) || defined(__thumb__) || defined(__arm__) || (defined(__riscv) && defined(__riscv_xlen) && (__riscv_xlen == 64)))
```

Comment:
Does this need the `__riscv_xlen` part of the logic test?  If you build the unix port on RV32 then it should also work with the native NLR.

Reply:
You are indeed correct, however consider that having Linux on RV32 is really uncommon.  I don't think I've ever seen an RV32 chip that can run a Linux environment for real work, short of a soft-core running on a high-end FPGA.

If you don't mind I'd leave this here and then provide a PR that handles both review points and brings RV32 under CI.

Reply:
OK.